modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/QueryAccessibleAttributeTest.java

```diff
@@ -24,40 +24,30 @@
  */
 package test.javafx.scene.control.skin;
 
-import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 import java.util.Collection;
-import java.util.List;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
 import javafx.scene.AccessibleAttribute;
 import javafx.scene.Node;
 import javafx.scene.control.Control;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 import test.com.sun.javafx.scene.control.infrastructure.ControlSkinFactory;
 
 /**
  * Tests whether queryAccessibleAttribute() in every Control handles all of the
  * AccessibleAttribute values without throwing an exception.
  */
-@RunWith(Parameterized.class)
 public class QueryAccessibleAttributeTest {
-    private Class<Node> nodeClass;
     private Node node;
 
-    @Parameterized.Parameters
-    public static Collection<Object[]> nodesUnderTest() {
-        List<Class<Control>> cs = ControlSkinFactory.getControlClasses();
-        return ControlSkinFactory.asArrays(cs);
+    private static Collection<Class<Control>> parameters() {
+        return ControlSkinFactory.getControlClasses();
     }
 
-    public QueryAccessibleAttributeTest(Class<Node> nodeClass) {
-        this.nodeClass = nodeClass;
-    }
-
-    @Before
-    public void setup() {
+    // @BeforeEach
+    // junit5 does not support parameterized class-level tests yet
+    public void setup(Class<Node> nodeClass) {
```

Comment:
Minor: Since `Control` is used above, should also be used here and below as Generic.

Reply:
hmm, the code is technically correct since Control extends Node.

Reply:
yes, just a minor nitpick since relaxing the bounds is not really needed